client/src/components/Ratings.jsx

```diff
@@ -1,23 +1,76 @@
+/* eslint-disable indent */
 import React from 'react';
-import Reviews from './Reviews';
+import ReviewsList from './ReviewsList';
+// Don't have to import the server/index.js?
 
-function Ratings() {
+
+
+// bring all reviews here to have data to go through
+
+// functions needed
+  // componentDidMount?
+
+  // calculate review to the nearest half star
+    // IMPORT Andrew's star function
+
+
+  // calculate percentage of reviews recommend product
+  // eslint-disable-next-line no-unused-vars
+  function getPercentageRecommended(ratingsArray) {
+
+  }
```

Comment:
try to avoid pushing empty functions to the main branch
use hardcoded data if you are still working on the logic